CHANGELOG.md

```diff
@@ -4,13 +4,41 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Common Changelog](https://common-changelog.org/).
 
+[1.2.0]: https://github.com/sablier-labs/v2-core/compare/v1.1.2...v1.2.0
 [1.1.2]: https://github.com/sablier-labs/v2-core/compare/v1.1.1...v1.1.2
 [1.1.1]: https://github.com/sablier-labs/v2-core/compare/v1.1.0...v1.1.1
 [1.1.0]: https://github.com/sablier-labs/v2-core/compare/v1.0.2...v1.1.0
 [1.0.2]: https://github.com/sablier-labs/v2-core/compare/v1.0.1...v1.0.2
 [1.0.1]: https://github.com/sablier-labs/v2-core/compare/v1.0.0...v1.0.1
 [1.0.0]: https://github.com/sablier-labs/v2-core/releases/tag/v1.0.0
 
+## [1.2.0] - 2024-06-16
+
+### Changed
+
+- **Breaking:** Move common logic into `Lockup` contract ([#813](https://github.com/sablier-labs/v2-core/pull/813),
+  [#850](https://github.com/sablier-labs/v2-core/pull/850), [#784](https://github.com/sablier-labs/v2-core/pull/784))
+- Disallow zero `startTime` ([#852](https://github.com/sablier-labs/v2-core/pull/852),
+  [#941](https://github.com/sablier-labs/v2-core/pull/941))
```

Comment:
this should be changed to: https://github.com/sablier-labs/v2-core/pull/813 since this PR introduces the removal of `startTime = 0` for linear

Reply:
Do you mean to include "#852 and #813" instead of "#852 and #941"?   Doesn't #941 also relevant to this changelog?

Reply:
>Do you mean to include "https://github.com/sablier-labs/v2-core/pull/852 and https://github.com/sablier-labs/v2-core/pull/813" instead of "https://github.com/sablier-labs/v2-core/pull/852 and https://github.com/sablier-labs/v2-core/pull/941"?

yes, this is what i mean

why would #941 be relevant here since it checks the start time against the `blockTimestamp`?

Reply:
https://github.com/sablier-labs/v2-core/pull/863/commits/8a03fba324caadecb46f73bcc4c3de649fac6ab7